pkgs/dart_tooling_mcp_server/test/test_harness.dart

```diff
@@ -187,8 +186,9 @@ final class AppDebugSession {
     while (vmServiceUri == null && await stdout.hasNext) {
       final line = await stdout.next;
       if (line.contains('A Dart VM Service')) {
-        vmServiceUri =
-            line.substring(line.indexOf('http:')).replaceFirst('http:', 'ws:');
+        vmServiceUri = line
+            .substring(line.indexOf('http:'))
+            .replaceFirst('http:', 'ws:');
```

Comment:
I've noticed formatting changes occurring frequently. Can we enforce formatting from the CI so that code is checked in with a consistent formatting? If possible, can we use the bleeding edge formatter from the flutter master branch?